README.md

```diff
@@ -1,4 +1,4 @@
-api-filter
+Api Filter
```

Comment:
Anywhere in docs (and composer description etc.) where there is "Api" or "api", it should be in capital letters  - "API Filter" etc.